manifests/2.15.0/opensearch-dashboards-2.15.0-test.yml

```diff
@@ -5,7 +5,111 @@ ci:
   image:
     name: opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4
 components:
-  - name: OpenSearch-Dashboards
+  - name: OpenSearch-Dashboards-ci-group-1
```

Comment:
I don't think this would throw error saying component not found as there is a mismatch.

Reply:
I think we are good from the python side as it only checks test manifest and gets list of components. If we can handle it from Jenkinsfile side as mentioned in my comment below then we are good. Right?